packages/lit-element/CHANGELOG.md

```diff
@@ -1,5 +1,55 @@
+- The dirty check in `attributeChangedCallback` has been removed. While technically breaking, in practice it should very rarely be ([#699](https://github.com/Polymer/lit-element/issues/699)).
+- LitElement's `adoptStyles` method has been removed. Styling is now adopted in `createRenderRoot`. This method may be overridden to customize this behavior.
+- LitElement's `static getStyles` method has been renamed to `static finalizeStyles` and now takes a list of styles the user provided and returns the styles which should be used in the element. If this method is overridden to integrate into a style management system, typically the `super` implementation should be called.
+- Removed build support for TypeScript 3.4.
+- Decorators are no longer exported from the `lit-element` module. Instead, import any decorators you use from `lit/decorators/*`.
+- `lit-html` has been updated to 2.x.
+- Support for running in older browsers has been removed from the default configuration. Import the `polyfill-support` module to support Shady DOM. Note also that Lit parts inside `<style>` elements are no longer supported. See [Polyfills](https://lit.dev/docs/tools/requirements/#polyfills) for more details.
+- For simplicity, `requestUpdate` no longer returns a Promise. Instead await the `updateComplete` Promise.
+- Removed `requestUpdateInternal`. The `requestUpdate` method is now identical to this method and should be used instead.
+- [#2103](https://github.com/lit/lit/pull/2103) [`15a8356d`](https://github.com/lit/lit/commit/15a8356ddd59a1e80880a93acd21fadc9c24e14b) - Updates the `exports` field of `package.json` files to replace the [subpath
+  folder
+  mapping](https://nodejs.org/dist/latest-v16.x/docs/api/packages.html#packages_subpath_folder_mappings)
+  syntax with an explicit list of all exported files.
+
+  The `/`-suffixed syntax for subpath folder mapping originally used in these
+  files is deprecated. Rather than update to the new syntax, this change replaces
+  these mappings with individual entries for all exported files so that (a) users
+  must import using extensions and (b) bundlers or other tools that don't resolve
+  subpath folder mapping exactly as Node.js does won't break these packages'
+  expectations around how they're imported.
+
+### Minor Changes
+
+- A public `renderOptions` class field now exists on `LitElement` and can be set/overridden to modify the options passed to `lit-html`.
+- Adds `static shadowRootOptions` for customizing shadowRoot options. Rather than implementing `createRenderRoot`, this property can be set. For example, to create a closed shadowRoot using delegates focus: `static shadowRootOptions = {mode: 'closed', delegatesFocus: true}`.
+- Adds development mode, which can be enabled by setting the `development` Node exports condition. See [Development and production builds](https://lit.dev/docs/tools/development/#development-and-production-builds) for more details.
```

Comment:
This seems like it should be much more prominent too, maybe the first item under minor changes?

Reply:
(Also I know this is just an automatic changesets thing, but I feel like the major/minor/patch headings aren't as good as the "added/changed/removed" etc. that we would usually have in a hand-rolled changelog. "minor changes" could easily be read as "things that aren't important enough for me to look at", but actually it's where the new features go :thinking: )

Reply:
(And I guess maybe for a big change like this we are expecting people to read the blog post, but personally when I notice some library I use has done even a major update, I always go straight for the changelog -- I don't think I have ever checked if they have a blog, twitter, etc.)

Reply:
(None of this blocking btw)